diff -ur osb-nrcit-0.5.0-o/src/HttpCurl.h osb-nrcit-0.5.0/src/HttpCurl.h
--- osb-nrcit-0.5.0-o/src/HttpCurl.h	2004-10-18 01:21:29.000000000 -0600
+++ osb-nrcit-0.5.0/src/HttpCurl.h	2005-10-29 19:08:52.000000000 -0600
@@ -78,7 +78,7 @@
     guint gsId; 
     
     /** NULL terminated list of supported protocols.  */
-    const char** curl_protocols; //not freed 
+    const char* const* curl_protocols; //not freed 
     
     /** proxy if one is set*/
     gchar * proxy;
